composer.json

```diff
@@ -21,11 +24,20 @@
         "oomphinc/composer-installers-extender": "^1.1 || ^2"
     },
     "config": {
+        "process-timeout": 0,
         "sort-packages": true
     },
     "extra": {
         "drupal-scaffold": {
+            "allowed-packages": [
+                "drupal/core",
```

Comment:
drupal/core is already implicitly allowed, explicitly listing it has no effect: https://github.com/drupal/core-composer-scaffold/blob/7b125516d6568b888945ee03ac2636dcced76e8d/AllowedPackages.php#L68-L69

Unfortunately the docs are not so good on this, I opened an upstream issue: https://www.drupal.org/project/drupal/issues/3221748

Reply:
Ah, thanks!